src/ConnectionWrapper.php

```diff
@@ -27,7 +27,7 @@
  *
  * This class is not tied with the Drupal Queue API.
  */
-class Connection {
+class ConnectionWrapper {
```

Comment:
The actual connection is stored inside the object instanciating this class, so the class was renamed for clarity. Elsewhere in the module, an instance of this class is stored in a `$connection_factory` variable whose name does not seem to suit with its actual purpose.